app/components/Buttons.tsx

```diff
@@ -0,0 +1,17 @@
+export default function PrimaryButton() {
```

Comment:
I would propose we have all file names lowercase! There is a convention where there is one React component per file. Since React components start with an uppercase letter, the file naming convention matched the React component names. 

However, these days I see much more usage of kebab-casing. Especially since we won't just have one component per file and the file name does not match the component names, I prefer kebab-casing. Also, npm package names follow kebab-casing as well.

Happy to discuss this next Monday!

Reply:
I've attempted to lowercase the file name but VS Code auto-corrects and capitalizes the first letter (maybe this is my personal settings - I'll continue to tinker with it). 
I have also created the file name "Components" where we can export all our other components -- is this what you mean by "one React component per file"?

Reply:
I meant, let's say we have a `ButtonLink`, a `Button`, and maybe an `IconButton` component. We can implement all three in the `buttons.tsx` file and export them. If we use `default` exports, then one of the buttons would be the default export while the other two aren't, which isn't super intuitive.

Reply:
> 

I see! File name has been changed to buttons.tsx